hls-test-utils/src/Test/Hls.hs

```diff
@@ -323,8 +326,14 @@ mkPluginTestDescriptor'
 mkPluginTestDescriptor' pluginDesc plId _recorder = IdePlugins [pluginDesc plId]
 
 -- | Initialise a recorder that can be instructed to write to stderr by
--- setting the environment variable "HLS_TEST_PLUGIN_LOG_STDERR=1" before
--- running the tests.
+-- setting the environment variable "HLS_TEST_PLUGIN_LOG_STDERR|LSP_TEST_LOG_STDERR
+-- |LSP_TEST_LOG_STDERR|HLS_TEST_LOG_STDERR=1" before running the tests.
```

Comment:
I think haddock will get confused by the `-- |`. 

Perhaps more lines make it better?

```suggestion
-- setting one of the environment variables:
--
-- * HLS_TEST_PLUGIN_LOG_STDERR=1
-- * HLS_TEST_LOG_STDERR=1
--
-- before running the tests.
```

I would omit 'LSP_TEST_LOG_STDERR', so that no one thinks about using it any more.